packages/components/typography/stories/typography.stories.js

```diff
@@ -0,0 +1,75 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '../lib/typography.cjs';
+
+storiesOf('ts-typography', module)
+	.add(
```

Comment:
We need to change those sub menu to variables in storybook

Reply:
Yeah, that's a great idea 👍